src/modules/editor/actions.ts

```diff
@@ -51,3 +51,35 @@ export const editorRedo = () => action(EDITOR_REDO, {})
 
 export type EditorUndoAction = ReturnType<typeof editorUndo>
 export type EditorRedoAction = ReturnType<typeof editorRedo>
+
+// Set Mode
+
+export const SET_MODE = 'Set mode'
+
+export const setMode = (mode: EditorMode) => action(SET_MODE, { mode })
+
+export type SetModeAction = ReturnType<typeof setMode>
+
+// Toggle Play
+
+export const TOGGLE_PREVIEW = 'Toggle preview'
+
+export const togglePreview = (enabled: boolean) => action(TOGGLE_PREVIEW, { enabled })
+
+export type TogglePreviewAction = ReturnType<typeof togglePreview>
+
+// Toggle Sidebar
+
+export const TOGGLE_SIDEBAR = 'Toggle sidebar'
+
+export const toggleSidebar = (enabled: boolean) => action(TOGGLE_SIDEBAR, { enabled })
+
+export type ToggleSidebarAction = ReturnType<typeof toggleSidebar>
+
+// Select Entity
+
+export const SELECT_ENTITY = 'Select entity'
+
+export const selectEntity = (entityId: string) => action(SELECT_ENTITY, { entityId })
```

Comment:
Is it going to be possible to deselect an entity?

Reply:
Good question, I don't know how the client works on that, but I suppose I could let the `entityId` also be `null` so we can `selectEntity(null)`

Reply:
Yes! or we could have a `deselectEntity()` with no params that calls `action(DESELECT_ENTITY, { entityId: null })`

Reply:
done ✅